lib/registrant.rb

```diff
@@ -0,0 +1,22 @@
+class Registrant
+
+    attr_reader :name, :age, :permit, :license_data
```

Comment:
Careful here, you have exposed a method called `.permit`, but we never need that method and don't actually want that method.